common-methods.go

```diff
@@ -113,8 +114,32 @@ func newClientFromAlias(alias string, urlStr string) (Client, *probe.Error) {
 	// We have a valid alias and hostConfig. We populate the
 	// credentials from the match found in the config file.
 	s3Config := new(Config)
-	s3Config.AccessKey = hostCfg.AccessKey
-	s3Config.SecretKey = hostCfg.SecretKey
+
+	// secretKey retrieved from the environement overrides the one
+	// present in the config file
+	KeysPairEnv := os.Getenv("MC_SECRET_" + alias)
```

Comment:
Yes, I didn't pay attention, you mean only the first character, right? 
By the way, gofmt didn't complain at all, do you you have any tool that detect that ?